README.md

```diff
@@ -748,6 +748,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [JADE](http://jade.tilab.com/) - Framework and environment for building and to debugging multi-agent systems.
 * [JavaVerbalExpressions](https://github.com/VerbalExpressions/JavaVerbalExpressions) - A library that helps to construct difficult regular expressions.
 * [Protégé](http://protege.stanford.edu/) - Provides an ontology editor and a framework to build knowledge-based systems.
+* [Underscore-java](http://javadev.github.io/underscore-java/) - A java port of Underscore.js. In addition to porting Underscore's functionality, Underscore-java includes matching unit tests.
```

Comment:
100% code coverage is a key feature of this library. I copied this text from github readme page.